src/content/documentation/develop/debugging.md

```diff
@@ -135,7 +135,9 @@ We use the [notify-link-clicks-i18n](https://github.com/mdn/webextensions-exampl
 :::
 
 <!-- Single Column Body Module -->
-[Background scripts](https://developer.mozilla.org/Add-ons/WebExtensions/Anatomy_of_a_WebExtension#Background_scripts) enable an extension to maintain a long-term state or perform long-term operations, independently of any web page or browser window. These scripts remain loaded for the lifetime of the extension. Background scripts are loaded inside an invisible background page: by default, this is an empty HTML document, but you can specify a custom page and define it as the background page using the `manifest.json` [`background`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/background) key.
+[Background scripts](https://developer.mozilla.org/Add-ons/WebExtensions/Background_scripts) enable an extension to monitor and react to events in the browser, such as navigating to a new page, removing a bookmark, or closing a tab. These scripts can be persistent or non-persistent. Persistent background scripts remain loaded for the lifetime of the extension. In contrast, non-persistent background scripts load when needed to respond to an event and unloaded when they become idle. Non-persistent background scripts are recommended because they consume less of the browser's resources. 
```

Comment:
`Non-persistent background scripts are recommended because they consume less of the browser's resources.` I'm not sure this page is the right place where to provide guidance about when a persistent vs. a non-persistent background page is required, or advised, without risking to have to repeat details that may be provided in another place where they fit better (e.g. in the API doc for the "background" manifest property).

e.g. we don't call out here that in MV3 only non-persistent background scripts are allowed, not that in MV2 the "persistent" property is still ignored.

I would propose to but out the last phrase (just the "Non-persistent background scripts are recommended ....").